easybuild/easyconfigs/a/Arrow/Arrow-8.0.0_bugfix.patch

```diff
@@ -0,0 +1,24 @@
+patch fixes a bug mentioned here: https://github.com/vscentrum/vsc-software-stack/issues/150#issuecomment-1862807311
+the bug is fixed in Arrow version 14: https://github.com/apache/arrow/pull/33802
```

Comment:
mention error explicitly:

```suggestion
fix for:
AttributeError: 'pyarrow.lib.BaseExtensionType' object has no attribute '__arrow_ext_class__'
see https://github.com/apache/arrow/pull/33802
```

keep in mind that changes the patch description implies updating the SHA256 checksum